src/main/java/seedu/address/logic/commands/FindByContactCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.person.ContactContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose contact contains any of the argument keywords.
+ */
+public class FindByContactCommand extends AbstractFindCommand {
+    public static final String COMMAND_WORD = "find /c";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose contact numbers "
+            + "contain any of the specified keywords (case-insensitive) and displays them as a list with indices.\n"
+            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
+            + "Example: " + COMMAND_WORD + " alice bob charlie";
```

Comment:
might consider changing the "alice bob charlie" to a number for a clearer message to the user

Reply:
true, but since test coverage is not part of the release we can keep this in view

Reply:
Thanks! Changed it with the new commit